### ISSUE TITLE
Disaster 2.14.3.1047

### ISSUE THREAD

**Issue by MtbGot** [ddvk/remarkable-hacks#474]
I wanted to update to the 1004 version, without knowing that the 1047 already existed...

So it updated, BUT I remembered somebody said that in case this happens DON'T restart...

I have blocked again auto updates... but I feel myself on the limbo....

**Comment by Eeems** [comment]
So what exactly are you asking for here? Is this just essentially a duplicate of #473 ?

**Comment by MtbGot** [comment]
> So what exactly are you asking for here? Is this just essentially a duplicate of #473 ?

Not exactly. He cancelled the update. In my case, it got installed while I was checking if version .1047 was already released... so I can't restart the device...

It would be amazing if the hack could be updated, but I understand it was already updated some weeks ago...

**Comment by Eeems** [comment]
You can switch back after updating: https://github.com/ddvk/remarkable-update/blob/main/switch.sh

**Comment by MtbGot** [comment]
Thank you!!!!!!

**Comment by evdp69** [comment]
I have the same issue as MtbGot using the RM2 and updating to 2.14.3.1047.
I tried to switch back using the procedure:
https://www.reddit.com/r/RemarkableTablet/comments/qftv1o/software_rollback_to_the_previous_version_only/

For some reason this procedure does not work for my RM2.
I get the message:
**new: 3
fallback: 2**
However after rebooting the software version is still 2.14.3.1047

Does somebody know what I do wrong, or is this RM1 procedure not working for RM2?

Thank you in advance for any assistance to solve this.

**Comment by evdp69** [comment]
> It's device independent. Did you run the script more than once before rebooting?

No, but I tried running the script three times, but always have rebooted after running the script. Two times with auto update of, and one time with auto update on.
I noticed indeed that I receive the message:
**new: 3
fallback: 2** ,
where in the instructions is stated:
**new: 2
fallback: 3**

 Any Idea how to solve this?

**Comment by Eeems** [comment]
> > It's device independent. Did you run the script more than once before rebooting?
> 
> No, but I tried running the script three times, but always have rebooted after running the script. Two times with auto update of, and one time with auto update on. I noticed indeed that I receive the message: **new: 3 fallback: 2** , where in the instructions is stated: **new: 2 fallback: 3**
> 
> Any Idea how to solve this?

The instructions just provide an example output. The second time you run the script you should see what the instructions provide. There are two root partitions, and this is just telling you which root partition it is switching to.

**Comment by evdp69** [comment]
> > > It's device independent. Did you run the script more than once before rebooting?
> > 
> > 
> > No, but I tried running the script three times, but always have rebooted after running the script. Two times with auto update of, and one time with auto update on. I noticed indeed that I receive the message: **new: 3 fallback: 2** , where in the instructions is stated: **new: 2 fallback: 3**
> > Any Idea how to solve this?
> 
> The instructions just provide an example output. The second time you run the script you should see what the instructions provide. There are two root partitions, and this is just telling you which root partition it is switching to.

OK, but in my case the software is not switched back to a former version.
Does somebody know a solution?
Could it be that something is not working because of a restriction?
Is it possible to reset the situation and try the switch back again?

**Comment by Eeems** [comment]
> OK, but in my case the software is not switched back to a former version.
> Does somebody know a solution?
> Could it be that something is not working because of a restriction?
> Is it possible to reset the situation and try the switch back again?

If both partitions have the same version installed, you wouldn't be able to switch.
There is nothing to reset here. You'd just need to use https://github.com/ddvk/remarkable-update to install a specific version.

**Comment by typO969** [comment]
> 
> Thank you for your assistance to help me. For me the procedure for installing a specific version is too difficult. I will wait till the next version release of ddvk. As I'm left handed it is almost impossible to work with the RM2 with standard software as I touch with my writing the (X) in the upper left corner. I have already send my complaints too Remarkable more than a year ago but they are not willing to solve the problem.

You're not alone, I am also a lefty and have sent multiple complaints about the UI to the company. Frustrating when program features turn into functionality killers.

**Comment by Tridens92** [comment]
Agreed, and I've also sent multiple messages over the years.  It's especially annoying because they advertise that the UI is lefty-friendly, and that they even have left-handers on staff.  Perhaps they don't use the device? ;)

**Comment by Eeems** [comment]
Just a reminder, this issue is about downgrading to a supported version. Lets stay on topic.

**Comment by MtbGot** [comment]
2.14.3.1047 is released!!!!
